tests/rt.conf

```diff
@@ -379,19 +379,21 @@ RUN     | fv3_ccpp_thompson_no_aero
 # CCPP PROD tests                                                                                                                                                                     #
 #######################################################################################################################################################################################
 
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta                                                                           | standard    | hera.intel     | fv3         |
+COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_2017_RRTMGP                                                       | standard    | hera.intel     | fv3         |
 
 RUN     | fv3_ccpp_gfs_v15p2                                                                                                             | standard    | hera.intel     | fv3         |
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | hera.intel     | fv3         |
+RUN     | fv3_ccpp_rrtmgp                                                                                                                | standard    | hera.intel     | fv3         |
 
 #######################################################################################################################################################################################
 # CCPP DEBUG tests                                                                                                                                                                     #
 #######################################################################################################################################################################################
 
-COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta DEBUG=Y                                                                   | standard    | hera.intel     | fv3         |
+COMPILE | CCPP=Y STATIC=Y SUITES=FV3_GFS_v15p2,FV3_GFS_v16beta,FV3_GFS_2017_RRTMGP DEBUG=Y                                               | standard    | hera.intel     | fv3         |
```

Comment:
Could we change all the rrtmgp test to be based on GFS_v16beta in later commits? We are going to update some of GFSv15 RT tests to run GFS_v16 configuration.

Reply:
Yes, certainly. Dustin did the original work based on fv3_ccpp_control.